xml/html_title.xml

```diff
@@ -4155,6 +4155,14 @@
     <param pos="0" name="service.product" value="R1Soft Server Backup Manager"/>
   </fingerprint>
 
+  <fingerprint pattern="^Transmission Web Interface$">
+    <description>Transmission</description>
+    <example>Transmission Web Interface</example>
+    <param pos="0" name="service.vendor" value="Transmission"/>
+    <param pos="0" name="service.product" value="Transmission"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:transmission:transmission:-"/>
```

Comment:
I overlooked this initially, but it appears the CPE was added by hand instead of following the [Updating CPEs](https://github.com/rapid7/recog/blob/main/CONTRIBUTING.md#updating-cpes) instructions since there doesn't appear to be a `a:transmission:transmission` in the NVD CPE dictionary. Transmission does have a CPE entry and it is `a:transmissionbt:transmission`. In this case, update all of the `service.vendor` parameters to `TransmissionBT`, follow the update CPE instructions, and commit the CPE updates disregarding any changes unrelated to the new fingerprints.